lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -132,7 +132,15 @@ test.serial('migratePdrRecord correctly migrates PDR record', async (t) => {
     testProvider,
   } = t.context;
   const executionPgModel = new ExecutionPgModel();
-  const execution = fakeExecutionRecordFactory();
+  const executionArn = 'arn:aws:states:us-east-1:12345:execution:test-IngestGranule:587f47b1-88b4-4f74-bd51-525085943fd3';
+  const executionURL = `https://console.aws.amazon.com/states/home?region=us-east-1#/executions/details/${executionArn}`;
```

Comment:
Not required, but we do have a `getExecutionUrlFromArn` helper in `packages/message/Executions` which might ensure some greater test stability

Reply:
Yup, good call: https://github.com/nasa/cumulus/pull/2390/commits/9d773f64b4d1d2aadb781e9b94759c70c67f3153